lib/host-sp-messages/src/lib.rs

```diff
@@ -507,6 +507,14 @@ pub enum InventoryData {
         id: [u8; 1024],
         temp_sensors: [SensorIndex; 2],
     },
+
+    /// W25Q256JVEIQ flash chip
+    W25q256jveqi {
+        mfr_id: u8,
+        memory_type: u8,
+        capacity: u8,
+        unique_id: [u8; 8],
+    }
```

Comment:
rustfmt would like a trailing comma here: https://github.com/oxidecomputer/hubris/actions/runs/15933362254/job/44947541537?pr=2119#step:4:25
```suggestion
    },
```